### PR TITLE
Fix image dimensions in Product Card view

### LINKED ISSUE #9738: [BOOM-4816] Cards dimensions don't match product images
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-4816).

- _**Reporter:**_ EmmanuelF
- _**Created at:**_ Tue, 13 Feb 2018 14:49:55 +0100

<p>On the catalog page of the front, the length of the cards is too big on both side (cf screenshots). </p>

<p> </p>

<p> </p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by CaptainYouz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR change the UI of the product image on the card view by removing margin.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9738 
| How to test?  | Check visual before/after

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10297)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Comment by marionf** [comment]
@CaptainYouz 

Before when the image was smaller we had this:

capture d ecran_287

Now, we have this:

capture d ecran_288

We should keep the previous behavior and center the image

**Review by PierreRambaud — DISMISSED** [review]

**Comment by marionf** [comment]
When the image is smaller, it' ok it's well centered

capture d ecran_846

But the original [issue](https://github.com/PrestaShop/PrestaShop/issues/9738) is not fixed, cards are still too big

capture d ecran_847

**Comment by PierreRambaud** [comment]
@marionf Should be ok now, I'm sorry :/

**Comment by marionf** [comment]
Yes, it's all good thank you @PierreRambaud

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl [hunk 1]
```diff
@@ -25,23 +25,23 @@
...
-            <img
-              src = "{$urls.no_picture_image.bySize.home_default.url}"
-            >
-          </a>
-        {/if}
-      {/block}
+      <div class="product-thumbnail">
+        {block name='product_thumbnail'}
+          {if $product.cover}
+            <a href="{$product.url}" class="thumbnail">
```
Comment (eternoendless) [inline themes/classic/templates/catalog/_partials/miniatures/product.tpl]: Changing the id breaks the tests (and potentially, other things). We should see if we can do this without changing the markup.

**Review by eternoendless — DISMISSED** [review]

**Comment by eternoendless** [comment]
Before PR:

Before

After PR:

After

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl (hunk 1 above)
Comment (PierreRambaud) [reply themes/classic/templates/catalog/_partials/miniatures/product.tpl]: There is no id here oO
We can update the xpath query.

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks everyone :)

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl (hunk 1 above)
Comment (eternoendless) [reply themes/classic/templates/catalog/_partials/miniatures/product.tpl]: I meant the class, sorry. I'm becoming more and more dyslexic as years go by 😁

**Pull request merged**